demonstrations/tutorial_vqe_uccsd_obs.py

```diff
@@ -0,0 +1,472 @@
+r"""
+VQE in different spin sectors with the Unitary Coupled Cluster ansatz
```

Comment:
```suggestion
VQE in different spin sectors
```
I know I suggested adding a reference to UCCSD, but after looking at the built docs I realize it is maybe too long and clunky